Source/WebCore/css/parser/CSSSupportsParser.h

```diff
@@ -1,5 +1,5 @@
 // Copyright 2015 The Chromium Authors. All rights reserved.
-// Copyright (C) 2016 Apple Inc. All rights reserved.
+// Copyright (C) 2016-2024 Apple Inc. All rights reserved.
```

Comment:
Not sure it’s right to be changing the date here but not otherwise touching the file in this pull request.